casepro/backend/junebug.py

```diff
@@ -463,3 +467,42 @@ def received_junebug_message(request):
         created_on=datetime.now(), has_labels=True)
 
     return JsonResponse(msg.as_json())
+
+
+@csrf_exempt
+def receive_identity_store_optout(request):
```

Comment:
Perhaps let's leave this for a follow up PR since it might get complicated. We'll probably need somewhere to set up and store credentials. Maybe `JUNEBUG_BACKEND`?